src/aks-preview/azext_aks_preview/tests/latest/test_validators.py

```diff
@@ -97,16 +102,19 @@ def test_one_empty_key_in_pair_one_non_empty(self):
 
     def test_invalid_key(self):
         cluster_autoscaler_profile = ["bad-key=val"]
-        namespace = Namespace(cluster_autoscaler_profile=cluster_autoscaler_profile)
+        namespace = Namespace(
+            cluster_autoscaler_profile=cluster_autoscaler_profile)
         err = "Invalid key specified for cluster-autoscaler-profile: bad-key"
 
         with self.assertRaises(CLIError) as cm:
             validators.validate_cluster_autoscaler_profile(namespace)
         self.assertEqual(str(cm.exception), err)
 
     def test_valid_parameters(self):
-        cluster_autoscaler_profile = ["scan-interval=20s", "scale-down-delay-after-add=15m"]
-        namespace = Namespace(cluster_autoscaler_profile=cluster_autoscaler_profile)
+        cluster_autoscaler_profile = [
+            "scan-interval=20s", "scale-down-delay-after-add=15m"]
+        namespace = Namespace(
```

Comment:
@IshanArya these ones still seem to have the funny formatting - can you quickly fix the formatting and then we are good to merge